client/src/document/hooks.ts

```diff
@@ -134,6 +134,16 @@ const sortByVisitsThenTimestampDesc = (
   return 0;
 };
 
+function getNextUniqueID(entries: FrequentlyViewedEntry[]): number {
```

Comment:
Since ids are usually unique, we can simplify the naming:

```suggestion
function getNextId(entries: FrequentlyViewedEntry[]): number {
```